packages/client/src/pages/game/components/stanbdy/standby.tsx

```diff
@@ -0,0 +1,115 @@
+import { useNavigate } from 'react-router-dom'
+import cn from 'classnames'
+
+import { GAME_STATES } from '@/pages/game/components/main'
+import TransparentButton from '@/components/ui/transparentButton/TransparentButton'
+import { ROUTES } from '../../../../routes'
+
+import styles from './index.module.scss'
+
+type Pops = {
+  gameState: GAME_STATES
+  onChangeState: (gameState: GAME_STATES) => void
```

Comment:
[Предложение]
Нравится оформление в TS для функций вот такое:
```suggestion
  onChangeState(gameState: GAME_STATES): void
```

Так они подсвечиваются другим цветом в ide, сразу видно, что функция, а что переменная

Reply:
изменила, спасибо))